code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/GithubOAuthImpl.java

```diff
@@ -41,7 +42,7 @@ public Account getAccount(ApiRequest request) {
             return null;
         }
         String token = request.getServletContext().getRequest().getHeader(AUTH_HEADER);
-        if (StringUtils.isEmpty(token)) {
+        if (StringUtils.isEmpty(token) || !token.startsWith(AUTH_TYPE)) {
```

Comment:
@vincent99 ok, I was confused exactly by the use case when "requests contain both a Authorization: Basic adsfasdf and ?token=jwtdata". I thought in this case the Authorization header should take precedence, and handling of such request should be delegated to BasicAuthImpl AccountLookup: